website/content/docs/agent-and-proxy/autoauth/methods/azure.mdx

```diff
@@ -16,7 +16,17 @@ on the value of the `resource` parameter.
 
 - `role` `(string: required)` - The role to authenticate against on Vault
 
-- `resource` `(string: required)` - The resource name to use when getting instance information
+- `authenticate_from_environment` `(boolean: false)` - If set to true, will attempt to make an Azure
+  identity object through Azure specific environment variables, in priority order and as specified [in
+  Azure documentation][environment-variables]. Will work seamlessly with Azure
+  [Workload Identity][workload-identity] and others. If set to true, `object_id`, and
+  `client_id` will be ignored in favour of this authentication approach.
+
+- `scope` `(string: optional)` - The scope to use when obtaining the token. If not given, will be
+  the value used for `resource` appended with `/.default` appended to it. For more information on
+  scopes in Azure, see [this document][scopes].
```

Comment:
```suggestion
- `scope` `(string: optional)` - The scope to use when obtaining a token. If not supplied, will default to 
  the value used for `resource` with "`/.default`" appended as a suffix. For more information on
  scopes in Azure, see [this document][scopes].
```

Reply:
Thanks! Will add this. Much better wording.